packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/resolution/ResolutionFeatureTest.java

```diff
@@ -329,4 +336,95 @@ public void computeBestPreviewSize_shouldUseQVGAWhenResolutionPresetLow() {
 
     mockedStaticProfile.verify(() -> CamcorderProfile.getAll("1", CamcorderProfile.QUALITY_QVGA));
   }
+
+  @Config(minSdk = 31)
```

Comment:
By creating this PR, I found out that these sorts of tests do not run with the current robolectric version, because it does not support configuring with SDK versions >= 31. I bumped it locally to verify these tests pass, but more work will have to be done to bump the robolectric version in the plugin. Filed https://github.com/flutter/flutter/issues/119752 for this issue.

Reply:
Thanks for filing that bug. Is your logic here that these tests probably pass and it is better to have them written and merged now even if they dont run than to wait until the issue is fixed and then write these tests? 

If so I think that makes sense. That said what is blocking this test from running?

Reply:
Yes, that's my reasoning. It's not running because it needs to run on SDK >= 31 to actually test the change and the current robolectric version does not support that.

Reply:
Ah so the mechanism is that robolectric is like "I have no idea what you are talking about so I will ignore this test"

Reply:
Right. Actually, if you set `sdk = 31` it gives an unknown SDK error, which I think is interesting. Keeping this issue on my radar so that we can get these tests running soon.